src/natcap/invest/habitat_quality.py

```diff
@@ -74,45 +74,38 @@
                 "LULC is provided, rarity outputs will be created for the "
                 "current and future LULC. The baseline LULC can have it's "
                 "own threat rasters (optional), where each threat raster "
-                "file path has a suffix of <b>_b</b>. If no threat rasters "
-                "are found, degradation and habitat quality outputs will not "
-                "be generated for the baseline LULC.<br/><br/> Each cell "
-                "should  represent a LULC code as an Integer.  The dataset "
-                "should be in a projection where the units are in meters and "
-                "the projection used should be defined. The LULC codes must "
-                "match the codes in the Sensitivity table.  If possible the "
-                "baseline map should refer to a time when intensive "
-                "management of the landscape was relatively rare."),
+                "file path is defined in the <b>Threats Data</b> CSV. "
+                "If there are no threat rasters and the threat paths are "
+                "left blank in the CSV column, degradation and habitat "
+                "quality outputs will not be generated for the baseline "
+                "LULC.<br/><br/> "
+                "Each cell should represent a LULC code as an Integer.  The "
+                "dataset should be in a projection where the units are in "
+                "meters and the projection used should be defined. The LULC "
+                "codes must match the codes in the Sensitivity table.  If "
+                "possible the baseline map should refer to a time when "
+                "intensive management of the landscape was relatively rare."),
             "name": "Baseline Land Cover"
         },
-        "threat_raster_folder": {
-            "validation_options": {
-                "exists": True,
-            },
-            "type": "directory",
-            "required": True,
-            "about": (
-                "A path to the directory that will contain all "
-                "threat rasters."),
-        },
         "threats_table_path": {
             "validation_options": {
-                "required_fields": ["THREAT", "MAX_DIST", "WEIGHT"],
+                "required_fields": [
+                    "THREAT", "MAX_DIST", "WEIGHT", "DECAY", "BASE_PATH",
+                    "CUR_PATH", "FUT_PATH"],
```

Comment:
Is `FUT_PATH` required?  Line 124 says that it's optional.

Reply:
Yea, I guess I'm straddling the fence here. It really is only required if future landcover is provided, however, for consistency I was thinking to say the column header was required but the values could certainly be empty. I feel having a static table format where headers are fixed and values can be left empty is a better and less confusing approach than allowing for columns to be present or absent but values being required if they are present. Ultimately a clear user message is the goal.

Reply:
@phargogh before updating docstrings, validation, User's Guide, I'll wait to hear your thoughts. For clarity I'm advocating for requiring all column headers to be present and to leave columns empty if not interested in a particular scenario. This would mean updating the Users's Guide and code documentation accordingly.

Reply:
> I feel having a static table format where headers are fixed and values can be left empty is a better and less confusing approach than allowing for columns to be present or absent but values being required if they are present. Ultimately a clear user message is the goal.

I agree, clear user feedback in case of error is critical regardless of the approach taken.

Well, this is a whole new tabular design for inputs to the model, so let's try it!  If we find it to be confusing to users or we're supporting a lot of people on the forums around this, then let's reconsider.